chrome_manifest_v3/class_app.js

```
@@ -0,0 +1,848 @@
+   */
+  saveSettings() {
+    const self = this;
+    const setID = self.CHROME_SETTINGS_ID;
+    let settings = Object.assign({}, self.popupView.data.settings);
+
+    // Delete large, potentially needing secure, data bits:
+    settings.description = '';
+    settings.title = '';
+    settings.attachments = [];
+    settings.images = [];
+    delete settings.description;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)